### ISSUE TITLE
Default shortcut in config gives out an error

### ISSUE THREAD

**Issue by galantra** [glutanimate/highlight-search-results#11]
#### Problem description

*Please describe the issue concisely in here. In case of an error: Walk us through the steps you took to get there. What happened? What did you expect to happen?*

1. go to the add-on's config window
2. click _restore config_
3. click OK

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x] I've restarted Anki to see if it helps
 - [x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [x] I've verified that I use the latest version of Anki by checking at https://apps.ankiweb.net#download
 - [ ] I've tried to disable other add-ons to see if there are any interactions present
 - [x] My issue disappears when I hold shift while starting Anki.
 - [x] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### Information about your Anki set-up

*Please open Anki, go to Help → About, click on "Copy Debug Info", and paste the result between the backticks below (if the button does not appear you are using an older version of Anki 2.1 and will need to update first):*

```
    
    Anki 2.1.53 (96bacf79) Python 3.9.7 Qt 5.15.2 PyQt 5.15.5
    Platform: Windows 10
    Flags: frz=True ao=True sv=2
    Add-ons, last update check: 2022-06-18 17:15:30
    
    
```
<details>
  <summary>
    ===Add-ons (active)===  </summary>
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    '' ['autoLearningSteps', 0, 'None', '']
    3ft Under Bury Newly Added Cards ['32017685', 2019-08-19T21:16, 'None', mod]
    AJT Mortician ['1255924302', 2021-09-17T07:52, 'None', mod]
    Add Table ['1237621971', 2022-04-12T17:21, 'None', mod]
    Adjust Sound Volume ['2123044452', 2021-05-05T13:24, 'None', mod]
    Advanced Answer Sounds ['1167194350', 2022-05-29T09:30, 'None', mod]
    Advanced Browser ['874215009', 2022-01-31T00:07, 'None', mod]
    Agent Orange Pseudo-Fuzz Defoliator ['175851166', 2020-11-20T06:21, 'None', '']
    Alternative Set Due Date Dialog for the Reviewer Touch-Friendly ['2107899486', 2022-02-23T22:51, 'None', mod]
    Anki Persistence ['1051511270', 2020-09-01T21:32, 'None', '']
    Anki Simulator ['817108664', 2021-08-07T21:24, 'None', '']
    Anki Zoom ['538879081', 2020-04-04T02:20, 'v1.1.3', mod]
    AnkiConnect ['2055492159', 2022-05-15T18:09, 'None', '']
    Asset Manager ['656021484', 2021-08-29T23:07, 'None', '']
    Auto Ease Factor ['1672712021', 2021-03-21T19:11, 'None', mod]
    Batch Editing ['291119185', 2019-08-08T19:56, 'None', '']
    BetterSearch ['1052724801', 2022-01-25T23:25, 'None', mod]
    Browser MaximizeHide TableEditorSidebar ['1819291495', 2022-01-25T23:05, 'None', '']
    Browser Resizer ['1435775540', 2021-08-01T23:11, 'None', '']
    Browser TableEditor side-by-side horizontal split ['831846358', 2022-01-25T23:11, 'None', mod]
    Bury Cousins related cards that arent siblings ['1072815885', 2021-07-20T03:23, 'None', '']
    Bury related notes ['413416269', 2020-04-03T09:55, 'None', mod]
    Butler-Addon ['1439937507', 2022-04-19T23:30, 'None', '']
    Card browser List only one card per note ['797076357', 2020-08-11T06:23, 'None', '']
    Cardistry 2 Dynamically Adjust New Cards ['1535078906', 2020-09-03T22:52, 'None', mod]
    Change Interface Font ['1431333984', 2021-08-27T02:36, 'None', mod]
    Change note type respect field and card names ['513858554', 2020-03-09T09:47, 'None', '']
    Cloze Overlapper for Anki 21 ['109531687', 2021-01-12T04:12, 'None', '']
    Colorful Browser Sidebar ['929609616', 2021-09-09T20:03, 'None', '']
    Correct Answers Graph ['993120073', 2022-03-14T21:52, 'None', '']
    Custom Background Image and Gear Icon ['1210908941', 2022-04-10T17:30, 'None', mod]
    Customize Keyboard Shortcuts ['24411424', 2022-05-13T08:39, 'None', mod]
    Deck name in title 21 ['699175524', 2019-08-23T23:06, 'None', '']
    Delay siblings ['1369579727', 2022-06-04T01:05, 'None', mod]
    Delete empty NEW cards tag other empty cards ['1402327111', 2021-10-16T19:41, 'None', mod]
    Dont remove mark on export ['909480379', 2020-02-16T10:53, 'None', '']
    Duplicate and Reorder ['1114271285', 2021-11-22T20:56, 'None', mod]
    Enhance main window ['877182321', 2020-12-23T05:42, 'None', '']
    Extended Schema Hash ['1187737890', 2020-01-06T18:25, 'None', '']
    External Editor ['16727059', 2020-07-27T11:06, 'None', mod]
    External Note Editor vim vscode atom ['704320065', 2022-05-17T14:42, 'None', mod]
    Fix Invalid Field HTML ['714624709', 2019-02-21T07:54, 'None', '']
    Flag label ['671965183', 2020-05-14T13:05, 'None', mod]
    Graphical form to change add-on configuration ['1014777615', 2020-03-19T11:22, 'None', '']
    HUMBLE PIE distinct cards studied today statistic ['2133933791', 2020-03-24T19:05, 'None', '']
    Hapax Predator - find and correct misspelings ['1495282885', 2022-01-20T01:07, 'None', '']
    Highlight Search Results in the Browser ['225180905', 2021-03-10T18:06, 'None', '']
    Holdem Cardfield Preserves add card history ['363320830', 2019-08-13T10:23, 'None', '']
    Image Occlusion Enhanced ['1374772155', 2022-04-09T09:15, 'None', '']
    Increase autosave frequency ['786971082', 2022-04-08T03:56, 'None', mod]
    Keep files git svn in add-on folders ['225953877', 2020-03-09T09:55, 'None', '']
    Load Balanced Scheduler ['208879074', 2019-08-08T20:32, 'None', '']
    Mini Format Pack ['295889520', 2019-08-07T21:49, 'None', mod]
    Minimal Flat Big Buttons compatible with ReMemorize Buttons ['1042429613', 2020-03-11T12:09, 'None', '']
    More Decks Stats and Time Left ['1556734708', 2022-05-20T12:23, 'None', '']
    More Overview Stats 21 ['738807903', 2021-07-30T21:13, 'None', '']
    No Distractions Full Screen Clean review interface  TabletTouch support ['1049863218', 2020-08-14T00:59, 'v4.1.8', mod]
    No Space Dash Prevents you from seeing answers without reading a question ['1409134430', 2020-12-25T09:49, '20.12.25i94', mod]
    OpenInExternalEditorRenameDuplicate for ImageAudioVideo ['1560623518', 2022-01-05T17:40, 'None', mod]
    Paste Images As WebP ['1151815987', 2022-06-01T22:54, 'None', '']
    Progress Graphs and Stats for Learned and Matured Cards ['266436365', 2020-03-29T08:26, 'None', '']
    Push Paradox Sibling Cardistry ['1797395052', 2020-04-04T06:40, 'None', '']
    ReColor ['688199788', 2022-03-02T17:11, '1.8', mod]
    Search Bar ['1251668918', 2020-09-09T23:55, 'None', '']
    Search in formatted text in any fields ['1126516755', 2021-10-16T19:37, 'None', mod]
    Small Potatoes Show learning step count in days ['75718778', 2020-04-02T20:53, 'None', '']
    Stats Overview Pie Graph with Distinct Learning and-or Relearning Sections ['1828603731', 2019-08-14T07:50, 'None', '']
    Tags are mandatory to add a card ['1116519045', 2020-05-02T12:17, 'None', '']
    Text Rendering Fix ['94394764', 2018-11-01T13:48, 'None', '']
    Throughput Historical ['485345973', 2019-08-18T09:28, 'None', '']
    Tippy Tooltips ['1840818335', 2022-04-10T17:50, 'None', '']
    Touch Wood Show New Cards Left Hint ['1498273991', 2020-04-04T06:37, 'None', '']
    Transfer scheduling data from one card to another ['94685914', 2020-09-18T11:23, 'None', '']
    True Retention by Card Maturity Simplified ['1779060522', 2020-06-28T15:45, 'None', '']
    True Tag Replacer ['803159801', 2021-02-28T11:07, 'None', '']
    add-on dialog searchfilter bar ['561945101', 2021-12-05T21:28, 'None', '']
    autoLapseNewInterval ['372281481', 2021-01-25T11:58, 'None', '']
    avgEase for Anki  2141 ['1129477042', 2021-06-27T13:58, 'None', '']
    confirm deletion ['263363568', 2019-04-13T13:21, 'None', '']
    copy note contents to new Add window from Browser Reviewer Add EditCurrent ['759018284', 2021-12-11T00:23, 'None', mod]
    deck and card info sidebar during review ['673114053', 2022-01-19T00:37, 'None', mod]
    deferpostpone card for later today selective bury and unbury ['1513254129', 2019-10-07T16:43, 'None', '']
    extended editor for field for tables searchreplace ['805891399', 2022-03-13T11:51, 'None', mod]
    remove linebreaks  do word wrap ['2028179508', 2021-12-11T00:35, 'None', mod]
    </details><details>  <summary>
    ===IDs of active AnkiWeb add-ons===  </summary>
    1014777615 1042429613 1049863218 1051511270 1052724801 1072815885 109531687 1114271285 1116519045 1126516755 1129477042 1151815987 1167194350 1187737890 1210908941 1237621971 1251668918 1255924302 1369579727 1374772155 1402327111 1409134430 1431333984 1435775540 1439937507 1495282885 1498273991 1513254129 1535078906 1556734708 1560623518 16727059 1672712021 175851166 1779060522 1797395052 1819291495 1828603731 1840818335 2028179508 2055492159 208879074 2107899486 2123044452 2133933791 225180905 225953877 24411424 263363568 266436365 291119185 295889520 32017685 363320830 372281481 413416269 485345973 513858554 538879081 561945101 656021484 671965183 673114053 688199788 699175524 704320065 714624709 738807903 75718778 759018284 786971082 797076357 803159801 805891399 817108664 831846358 874215009 877182321 909480379 929609616 94394764 94685914 993120073  </details>
    <details>
  <summary>
    ===Add-ons (inactive)===  </summary>
    (add-on provided name [Add-on folder, installed at, version, is config changed])
    '' ['1135507716 bettertags', 0, 'None', mod]
    '' ['1135507716 tagedit_subtag_completer', 0, 'None', '']
    '' ['No Penalties or Boosting', 0, 'None', '']
    '' ['PassFail (EN)', 0, 'None', '']
    '' ['ResetEZ', 0, 'None', '']
    '' ['more_answer_buttons', 0, 'None', mod]
    '' ['pez_dispenser', 0, 'None', '']
    '' ['roommate_switch', 0, 'None', '']
    '' ['show_warning_if_no_tags_when_adding_cards', 0, 'None', '']
    '' ['visual_feedback', 0, 'None', '']
    '' ['wordsworth', 0, 'None', '']
    3 add-ons merged quicker anki explain deletion explain database check ['777545149', 2019-09-28T08:05, 'None', '']
    Add Parent Tag To Every Tag In A Deck ['543106186', 2019-11-21T13:20, 'None', '']
    Add a tag to notes with missing media ['2027876532', 2022-06-05T02:28, 'None', '']
    Advanced Card Styles ['1625103157', 2021-12-28T18:49, 'None', '']
    Advanced Copy Fields ['1898445115', 2019-08-17T11:52, 'None', '']
    Age Updater ['772210088', 2021-08-31T20:53, 'None', '']
    Allows empty first field during adding and import ['46741504', 2021-10-17T23:27, 'None', '']
    Anki Habitica for 21 ['1758045507', 2022-02-09T13:54, 'None', '']
    Anki Killstreaks ['579111794', 2021-12-15T16:20, 'None', mod]
    Anki with Friends ['613520216', 2022-02-05T17:40, 'None', mod]
    Ankimote - remote to control Anki from your phone ['149004221', 2021-08-11T18:57, 'None', '']
    Answer Button Sounds ['679615590', 2020-02-20T16:10, 'None', '']
    Answer visual  auditory  custom text reinforcement ['976516370', 2021-10-24T17:05, 'None', '']
    Audio Files ['1984823157', 2022-04-18T19:03, 'None', mod]
    AutoDefine - Automatically define vocabulary words with pronunciations and image ['2136497008', 2020-11-16T00:16, 'None', mod]
    AwesomeTTS - Add speech to your flashcards ['1436550454', 2022-05-21T02:28, 'None', '']
    AwesomeTTS Google Cloud Text-to-Speech  Microsoft Azure TTS unofficial ['814349176', 2021-06-16T23:08, 'None', '']
    Batch Download Pictures From Google Images ['561924305', 2020-07-09T14:51, 'None', '']
    Browser Card Info Bar ['2140680811', 2022-01-25T23:15, 'None', '']
    CTRLF5 to Refresh the Browser ['1347728560', 2019-08-08T20:58, 'None', '']
    Card Info During Review ['2179254157', 2022-04-15T07:26, 'None', '']
    Change Order of Review Cards in Regular Decks ['3731265543', 2019-11-25T23:30, 'None', '']
    Change cards decks prefix ['1262882834', 2019-08-10T10:14, 'None', '']
    Clean the default deck ['683170394', 2019-12-07T17:03, 'None', '']
    Clickable Tags v20 original ['380714095', 2022-01-30T23:58, 'None', '']
    Closet For Anki ['272311064', 2021-04-12T16:09, 'None', '']
    Cloze Overlapper ['cloze_overlapper', 2019-02-01T13:37, 'None', '']
    Collapsible Fields ['1896168623', 2020-12-22T03:09, 'None', '']
    Color Confirmation ['1084228676', 2020-03-14T13:13, 'None', mod]
    Colorful toolbars ['388296573', 2019-11-21T13:23, 'None', '']
    Content Injector ['166455199', 2020-09-03T18:08, 'None', '']
    Control Audio Playback Extended ['1413404470', 2020-02-12T12:57, 'None', '']
    Convert AudioBooks to Anki WIP ['1885548963', 2020-05-29T23:10, 'None', '']
    Convert Subdecks to Tag Hierarchy ['1172858842', 2020-02-23T23:03, 'None', '']
    Copy notes ['1566928056', 2021-12-24T20:56, 'None', mod]
    Create link in note to display specific search  in browser ['2053932397', 2020-03-29T19:56, 'None', '']
    Cross Profile Search and Import ['310394744', 2019-08-10T10:00, 'None', '']
    CrowdAnki JSON exportimport Edit history Collaborate on deck creation ['1788670778', 2022-05-27T23:32, 'None', mod]
    Customize Editor Stylesheet ['1215991469', 2019-11-23T11:38, 'None', '']
    Customize Sidebar ['1988760596', 2020-01-15T22:01, 'None', mod]
    Deadline2 ['723639202', 2021-05-02T06:45, 'None', mod]
    Deck Counts NowLater ['1836212767', 2020-09-09T17:33, 'None', '']
    Directly review without going through overview page ['1024346707', 2019-11-07T09:36, 'None', '']
    Do not underline current queue in reviewer ['267531099', 2019-08-17T12:23, 'None', '']
    Ease Factor Histogram ['494944895', 2019-08-07T22:57, 'None', '']
    Extended Card Stats During Review ['1008566916', 2019-08-15T10:43, 'None', '']
    Extended Tag AddEdit Dialog ['1135507717', 2022-03-13T11:41, 'None', mod]
    External Note Editor for the Browser ['1996266159', 2020-03-31T22:59, 'None', '']
    Fast Word Query Multi-threaded queries for words from local or web dictionaries ['1807206748', 2019-08-07T22:16, 'None', '']
    Fastbar- with nightmode support ['46611790', 2022-04-15T05:04, 'None', '']
    Field History ['1247884413', 2019-08-15T11:03, 'None', '']
    Fill the blanks - Multiple typecloze support ['1933645497', 2021-08-05T21:38, 'None', '']
    Find Missing Words ['754868802', 2019-11-18T07:34, 'None', '']
    Find words and create deck ['1572743214', 2020-02-12T12:57, 'None', '']
    Fluent Forever - Open Websites ['143682703', 2019-12-22T18:22, 'None', '']
    Forked Improved Quzlet to Anki Importer english ['1302146328', 2019-11-24T07:56, 'None', '']
    High Yield Tags ['1918380616', 2021-08-07T02:48, 'None', mod]
    Hitmarkers ['1776869150', 2020-04-01T00:04, 'None', '']
    HoochieBaby Queue Controller  Day-Lrn Q ['1847358755', 2020-11-23T03:54, 'None', '']
    HoochieMama Randomize Rev Queue ['1460733408', 2020-06-03T15:17, 'None', '']
    HoochiePapa Randomize New Queue ['1173108619', 2020-04-25T05:17, 'None', '']
    If a note has no more card warns instead of deleting it ['2018640062', 2019-08-13T10:19, 'None', '']
    Ignore accents in browser search ['1924690148', 2020-11-28T12:29, 'None', '']
    Improved Quizlet to Anki 21 Importer ['538351043', 2020-11-15T03:16, 'None', '']
    Incremental Reading v4103 ['935264945', 2019-08-10T18:13, 'None', '']
    Interactive Cards for language learning ['1819109876', 2019-08-07T23:35, 'None', '']
    Keep card displayed in reviewer until it is modified ['1894745652', 2020-01-24T02:22, 'None', '']
    Keep model of add cards ['424778276', 2021-10-16T19:36, 'None', '']
    Kitten Rewards Gamification ['1627107763', 2018-11-10T09:14, 'None', '']
    Knockout Deck Subset Learning ['899747926', 2019-08-09T09:44, 'None', '']
    Leaderboard ['41708974', 2022-05-12T11:46, 'None', mod]
    Learning Step and Review Interval Retention ['1949865265', 2022-03-14T21:58, 'None', '']
    Life Drain ['715575551', 2022-02-13T14:15, 'None', '']
    Limit new cards based on due  reviews ['548158563', 2021-06-09T08:26, 'None', mod]
    Long term backups ['529955533', 2019-07-08T04:37, 'None', '']
    Migaku Retirement ['1666520655', 2021-01-14T01:37, 'None', '']
    More Answer Buttons with colors ['1550991760', 2019-10-28T10:24, 'None', '']
    MorphMan for Anki 21 ['900801631', 2021-02-11T05:30, 'None', '']
    Multi-column note editor ['3491767031', 2021-11-02T16:18, 'None', mod]
    Newline in strings in add-ons configurations ['112201952', 2020-03-19T19:41, 'None', '']
    Nonconsecutive Search in Study Deck ['984507853', 2020-01-04T20:09, 'None', '']
    Note Comparer ['188442097', 2021-09-17T17:40, 'None', '']
    Number of Decks ['1358364553', 2019-12-11T20:01, 'None', '']
    Open Added Today from Reviewer ['861864770', 2019-12-07T17:03, 'None', '']
    Opening the same window multiple time ['354407385', 2020-05-16T02:59, 'None', '']
    Partial synchronization for inconsistent database ['1675325364', 2020-03-06T06:33, 'None', '']
    Pennywise Add Only Once Option ['1032766035', 2019-08-13T10:24, 'None', '']
    Polar Connect - PDF and Incremental Reading with Anki Sync ['734898866', 2019-08-07T22:36, 'None', '']
    Pomodore Tomato Clock Port ['811976365', 2019-08-25T19:07, 'None', '']
    Pop-up Dictionary beta ['153625306', 2019-08-09T18:26, 'None', mod]
    Pop-up Wikipedia Beta ['395343016', 2020-01-14T00:26, 'None', '']
    Postpone cards review ['1152543397', 2021-01-29T00:20, 'None', '']
    Pretzel Logic Retention Benchmarking ['1168065472', 2020-04-25T05:13, 'None', '']
    Progress Bar ['2091361802', 2017-08-29T21:00, 'None', '']
    Puppy Reinforcement ['1722658993', 2020-03-03T08:49, 'None', '']
    Quantified Self add-on - export your review log fork for Anki 21 ['1806723271', 2020-10-30T11:09, 'None', '']
    Quick note and deck buttons Fork for 21 ['1331140127', 2022-05-23T15:13, 'None', mod]
    Quick tagging 21 ['304770511', 2020-02-11T07:17, 'None', mod]
    Random Sprites ['897478712', 2022-01-24T08:38, 'None', '']
    ReMemorize Buttons ['rememorize_btn', 2019-04-30T09:32, 'None', mod]
    Rebuild All  Empty All for Anki 21 ['1810938259', 2019-11-14T07:56, 'None', '']
    Redesign ['1914733489', 2020-02-29T23:46, 'None', '']
    Refocus Card when Reviewing 21 ['1642550423', 2019-08-07T23:20, 'None', '']
    Reloadbackup menu option ['883617253', 2020-02-10T19:06, 'None', '']
    Remaining time for Anki 21 ['1508357010', 2020-12-13T15:14, 'None', '']
    Remove card history ['2089200096', 2018-04-23T08:50, 'None', '']
    Removes Empty Note Types for 21 ['530953383', 2019-08-17T08:30, 'None', '']
    Replay buttons on card ['498789867', 2019-08-07T22:10, 'None', '']
    Reset Ease ['947935257', 2020-12-27T15:59, 'None', '']
    Review Heatmap ['review_heatmap', 2020-04-30T15:23, 'None', '']
    Reviewer Context Menu Search ['359618071', 2019-08-19T14:31, 'None', '']
    Reviewer to Browser choosing what to display ['1555020859', 2020-12-23T05:58, 'None', '']
    Right Hand Answer Shortcuts ['1816717670', 2020-03-14T19:45, 'None', '']
    Search and Replace Tags ['138501288', 2019-09-12T10:23, 'None', '']
    Separate Learn and Relearn in the Answer Buttons graph 21 ['403491806', 2019-11-10T08:55, 'None', '']
    Set Randomizer v22 ['1164532380', 2019-12-01T21:33, 'None', mod]
    Show fields source html in editor ['1574324795', 2021-10-16T19:39, 'None', '']
    Slackers Delight PostponeDefer Button ['880824023', 2020-02-27T20:59, 'None', '']
    Special Fields ['1102281552', 2022-02-23T18:27, 'None', mod]
    Sticky Tags ['1433159067', 2021-12-17T12:52, 'None', '']
    Straight Reward ['957961234', 2022-03-31T21:37, 'None', '']
    Syllabus - Anki Statistics and Export by Tag and Deck ['668156440', 2020-01-12T12:47, 'None', mod]
    Symbols ['1461102936', 2019-08-11T21:28, 'None', mod]
    SyncRedirector ['2124817646', 2020-04-07T08:10, 'None', '']
    Tag Entry Enhancements v2 ['536796161', 2020-11-10T12:28, 'None', '']
    TagSelectorV2 ['1022577188', 2021-08-07T13:28, 'None', '']
    Templates Import  Export ['712027367', 2022-04-13T17:26, '0.2', '']
    Time Spent Creating Cards ['1178407876', 2020-07-01T22:08, 'None', '']
    Time per revised card graph ['1703703955', 2021-01-27T01:12, 'None', '']
    Web Browser - Search terms Import texts and images automatically ['864545277', 2022-01-17T22:33, 'None', mod]
    Web Query  ['627484806', 2019-08-11T15:00, 'None', '']
    Ze Add Note Id ['886709118', 2021-03-02T21:06, 'None', '']
    Zoom 21 ['1846592880', 2020-01-13T10:16, 'None', mod]
    add and bury show later ['1434066995', 2020-03-06T01:06, 'None', mod]
    ankisyncd new ['80439555', 2020-05-10T08:05, 'None', '']
    avgEase ['633293039', 2018-11-14T18:21, 'None', '']
    browser start on topscroll to top in editor ['1975943105', 2020-06-10T23:26, 'None', '']
    ease modify extended ['1124029139', 2019-11-21T13:21, 'None', '']
    editor batch replace in one field also with regex ['356095608', 2021-12-18T21:33, 'None', '']
    export all notetypesmodels and deckoptionsgroups without notes or media ['870620435', 2019-11-23T11:34, 'None', '']
    export cardsnotes from browser with metadata to csv or xlsx ['1967530655', 2022-04-12T17:17, 'None', '']
    scheduler Dynamic Lapse Interval alpha ['678594073', 2020-08-14T11:15, 'None', '']
</details>

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

```

There was a problem with the provided configuration: 'Ctrl-*- T, H’ does not match 'A((([Cc][Tt][Rr][LI]|[Aa][LI][Tt]|[Ss][Hh][li][Ff][Tt])\\+){0,3}([a-zA-Z0-9]|F[1-9]| F1[0-2]))?S', at path hotkey_toggle_highlights, against schema {'type': 'string', 'title': 'Hotkey to toggle highlights on/off', 'pattern':'A((([Cc][Tt][Rr][LI]|[Aa][LI] [Tt]|[Ss][Hh][li][Ff][Tt])\\+){0,3}([a-zA-Z0-9]|F[1-9]|F1[0-2]))?S'}.
```

![imagen](https://user-images.githubusercontent.com/52420923/174467028-573c4264-68d5-4872-bb20-3f2600d1c516.png)
